sdk/master/api_admin.go

```diff
@@ -479,6 +479,16 @@ func (api *AdminAPI) GetVolumeSimpleInfo(volName string) (vv *proto.SimpleVolVie
 	return
 }
 
+func (api *AdminAPI) SetVolumeForbidden(volName string, forbidden bool) (err error) {
+	request := newAPIRequest(http.MethodGet, proto.AdminVolForbidden)
+	request.addParam("name", volName)
+	request.addParam("enable", strconv.FormatBool(forbidden))
```

Comment:
it may be better to change var name `enable` to `enableForbidden` or `forbiden`.